test/qtismtest/data/UtilsTest.php

```diff
@@ -147,4 +148,23 @@ public function testgetLastItem2()
         $this->assertEquals(null,
             DataUtils::getLastItem($test, $test->getComponentByIdentifier('S96'), $sections));
     }
+
+    public function testSanitizeIdentifier()
```

Comment:
In order to avoid non necessary loops, could you please use the [dataprovider](https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers) technique ? This way more elegant!